packages/subgraph/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "maci-subgraph",
+  "version": "2.2.1",
```

Comment:
we released 2.3.0 on Friday btw. Also, are we gonna keep both subgraphs or only one? what you think?

Reply:
Yes, I forgot to change these fields.

Reply:
I think for now we can keep both subgraphs because they can be reused. `maci` has a template and `maci-platform` has an application.